ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>Cloud SDK -&gt; Google Cloud SDK</p><p> </p><p>version -&gt; resource</p><p> </p><p>It must be unique across all &nbsp;Cloud Storage buckets. -&gt; It must be unique across all &nbsp;Cloud Storage buckets in your project.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done except Cloud SDK</p>